### DISCUSSION TITLE
Comment récupérer le Token Enedis ?

### DISCUSSION THREAD

**Discussion by GooodFr** [bokub/conso-api#53] [Aide] [answered]
Bonjour,

Je suis très intéressé de lire mes données Linky.
J'ai installé hyper facilement un Home Assistant sur un Pi4
J'ai installé l'Add-on Linky Facilement

Ce que je ne comprends pas, c'est comment obtenir un Token chez Enedis.
J'ai suivi plusieurs tutos qui mènent tous à des impasses :(

J'ai activé la collecte de la consommation horaire, mais je ne peux rien obtenir d'autre sur le site Enedis.
J'ai rempli un compte sur Datahub-enedis.fr (wow … )
Faut que je souscrive à Data Connect !?
Je l'ai fais 2 fois, mais j'ai pas l'impression d'être sur la bonne piste …

**Comment by bokub** [comment, marked as answer]
Salut,

En effet tu n'est pas du tout sur la bonne piste. Puis-je te demander quels tutoriels tu as suivi ?

Si je comprend bien tes messages, tu veux un token pour faire fonctionner mon add-on HA Linky. Dans ce cas je t'invite à lire uniquement le guide de [HA Linky](https://github.com/bokub/ha-linky#readme), dans lequel tu verras écrit noir sur blanc qu'il te faut simplement générer un token [**Conso API**](https://conso.boris.sh/)

**Comment by GooodFr** [comment]
Oulala ! Ca y est, cela fonctionne.
C'est justement ce que j'ai fait en premier.

Et sur la page Conso API, aujourd'hui, le lien ne me renvois pas au même endroit !
Je devais faire une demande "collecte de la consommation horaire"
Aujourd'hui; j'ai eu mon token :D

J'ai manqué de persévérance, désolé ;)
Merci ! :)
Pour la réponse, et pour l'API !!!

**Reply by bokub** [reply]
Ah mince, j'imagine qu'Enedis avait cassé la redirection vers la page de consentement :shrug: 

Cool que tu aies réussi !

**Reply by Lyric76** [reply]
Bonjour, depuis un semaine , j'essaie en vain de voir comment récupérer le token sur le site enedis en suivant ta documentation et tes liens. Mon objectif est de l'utiliser dans le module home assistant. J'ai fait les intégrations HA. J'ai créé mon compte enedis, je vois mes conso et mes consentements existants EDF et hellowatt. A aucun moment je ne vois où on me propose de récupérer ma clé. Le site d'enedis était en maintenance plusieurs jours, auraient ils changer des liens ?

**Reply by bokub** [reply]
Hello,

Tu trouveras ta réponse dans #62
